openwisp_firmware_upgrader/admin.py

```diff
@@ -64,6 +65,10 @@ class BuildAdmin(BaseVersionAdmin):
     inlines = [FirmwareImageInline]
     actions = ['upgrade_selected']
     multitenant_parent = 'category'
+    change_form_template = 'admin/firmware_build/change_form.html'
```

Comment:
you don't need to do this, just place the template at the following path and this will not be needed:
`admin/firmware_upgrader/build/change_form.html`.

[This mechanism is well documented in the django docs](https://docs.djangoproject.com/en/3.0/ref/contrib/admin/#set-up-your-projects-admin-template-directories).